### ISSUE TITLE
Console error spam

### ISSUE THREAD

**Issue by Prof1611** [SilverstoneMC/EntityClearer#36]
### Description of the bug

Console is constantly spammed with errors on the newest release of the plugin:
```
[14:08:59] [Server thread/WARN]: [EntityClearer] Task #65972 for EntityClearer v3.7.0 generated an exception
java.lang.NoSuchMethodError: 'int org.bukkit.World.getPlayerCount()'
        at net.silverstonemc.entityclearer.utils.OnlinePlayers.isNotEnough(OnlinePlayers.java:28) ~[?:?]
        at net.silverstonemc.entityclearer.Countdown.message(Countdown.java:63) ~[?:?]
        at net.silverstonemc.entityclearer.Countdown$1.run(Countdown.java:45) ~[?:?]
        at org.bukkit.craftbukkit.v1_20_R3.scheduler.CraftTask.run(CraftTask.java:82) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at org.bukkit.craftbukkit.v1_20_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:415) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1339) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:387) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1242) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1054) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4060-Spigot-e9ec548-366c3ca]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### Steps to reproduce the problem

Run the plugin, I'm on Spigot 1.20.4

### Expected behavior

The plugin ran normally

### Output of `/version`

[14:13:19] [Server thread/INFO]: This server is running CraftBukkit version 4060-Spigot-e9ec548-366c3ca (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT)

### Output of `/version EntityClearer`

[14:13:25] [Server thread/INFO]: EntityClearer version 3.7.0
[14:13:25] [Server thread/INFO]: Remove entities from your worlds!
[14:13:25] [Server thread/INFO]: Website: https://github.com/SilverstoneMC/EntityClearer
[14:13:25] [Server thread/INFO]: Author: JasonHorkles

### Debug file

```yaml
[14:14:05] [Server thread/INFO]: Activating debug... See console for more details.
[14:14:05] [Server thread/INFO]: [EntityClearer] Dumping config into debug file...
[14:14:05] [Server thread/INFO]: [EntityClearer] Config file dumped!
[14:14:05] [Server thread/INFO]: [EntityClearer] 
[14:14:05] [Server thread/INFO]: [EntityClearer] ????????????????????????????????????????
[14:14:05] [Server thread/INFO]: [EntityClearer] ?             INFORMATION              ?
[14:14:05] [Server thread/INFO]: [EntityClearer] ????????????????????????????????????????
[14:14:05] [Server thread/INFO]: [EntityClearer] Plugin version: 3.7.0
[14:14:05] [Server thread/INFO]: [EntityClearer] Server version: CraftBukkit 4060-Spigot-e9ec548-366c3ca (MC: 1.20.4)
[14:14:05] [Server thread/INFO]: [EntityClearer] API version: 1.20.4-R0.1-SNAPSHOT
[14:14:05] [Server thread/INFO]: [EntityClearer] Java version: 17.0.7
[14:14:05] [Server thread/INFO]: [EntityClearer] Available world list: 
[14:14:05] [Server thread/INFO]: [EntityClearer]  world
[14:14:05] [Server thread/INFO]: [EntityClearer]  world_nether
[14:14:05] [Server thread/INFO]: [EntityClearer]  world_the_end
[14:14:05] [Server thread/INFO]: [EntityClearer]  lobby
[14:14:05] [Server thread/INFO]: [EntityClearer]  HalloweenMansion
[14:14:05] [Server thread/INFO]: [EntityClearer]  lonehill
[14:14:05] [Server thread/INFO]: [EntityClearer]  creative
[14:14:05] [Server thread/INFO]: [EntityClearer]  witp
[14:14:05] [Server thread/INFO]: [EntityClearer] 'ALL' found! Adding all worlds to list...
[14:14:05] [Server thread/INFO]: [EntityClearer] 
[14:14:05] [Server thread/INFO]: [EntityClearer] ????????????????????????????????????????
[14:14:05] [Server thread/INFO]: [EntityClearer] ?     REMOVE ENTITIES TASK STARTED     ?
[14:14:05] [Server thread/INFO]: [EntityClearer] ????????????????????????????????????????
[14:14:05] [Server thread/INFO]: [EntityClearer] 
[14:14:05] [Server thread/ERROR]: [EntityClearer] SERVER: Something went wrong clearing entities! Is your config outdated?
[14:14:05] [Server thread/ERROR]: [EntityClearer] java.lang.NoSuchMethodError: 'int org.bukkit.World.getPlayerCount()'
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.silverstonemc.entityclearer.utils.OnlinePlayers.isNotEnough(OnlinePlayers.java:28)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.silverstonemc.entityclearer.ClearTask.removeEntities(ClearTask.java:72)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.silverstonemc.entityclearer.ClearTask.removeEntitiesPreTask(ClearTask.java:46)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.silverstonemc.entityclearer.Commands.debug(Commands.java:129)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.silverstonemc.entityclearer.Commands.onCommand(Commands.java:29)
[14:14:05] [Server thread/ERROR]: [EntityClearer] org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[14:14:05] [Server thread/ERROR]: [EntityClearer] org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149)
[14:14:05] [Server thread/ERROR]: [EntityClearer] org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchCommand(CraftServer.java:887)
[14:14:05] [Server thread/ERROR]: [EntityClearer] org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchServerCommand(CraftServer.java:872)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.minecraft.server.dedicated.DedicatedServer.bk(DedicatedServer.java:412)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:388)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1242)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1054)
[14:14:05] [Server thread/ERROR]: [EntityClearer] net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304)
[14:14:05] [Server thread/ERROR]: [EntityClearer] java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by JasonHorkles** [comment]
I'll look into it. In the meantime, it should work fine on Paper if you need a quick fix

**Comment by JasonHorkles** [comment]
Can you please try this and see if it works? If so I'll push the release
[EntityClearer.zip](https://github.com/user-attachments/files/17534632/EntityClearer.zip)

**Comment by Prof1611** [comment]
Will give it a try, it seemed to take a while to start spamming with errors so will have to report back in a day or so. Thanks!

**Comment by JasonHorkles** [comment]
I presume you haven't come across more errors?

**Comment by Prof1611** [comment]
All seems good!

**Comment by JasonHorkles** [comment]
Closed via 9a382de
